website/static/js/fangorn.js

```diff
@@ -1009,6 +1020,34 @@ function _removeEvent (event, items, col) {
     }
 }
 
+function doCheckout(item, checker, error_message) {
+    $.ajax({
```

Comment:
Correction: you cannot currently pass `isCors` to `$osf.putJSON` . You can, however, pass it to `ajaxJSON` (h/t to @abought for the clarification).